### PR TITLE
Add test case for virsh domfsfreeze

### PR DESCRIPTION
**Pull request by liuzzfnst**

### CODE REVIEW

**Comment by PandaWei** [comment]
Hi,
This PR has the same problem as #568 .
Please fix it and attach your test result

**Comment by PandaWei** [comment]
I have no other suggestions about this PR.
but show us the result.
Thanks

**Comment by PandaWei** [comment]
@liuzzfnst 
Hi, 
there're some testcases about domfsfreeze. #427 
So, Please take a look at it, and found if it has any difference。
if not, i suggest you close this PR.

Maybe we should check the PR list before pushing next time.

**Comment by liuzzfnst** [comment]
@PandaWei 
I think a acl test case should be added into #427, I will fix my patches according to #427.

**Comment by PandaWei** [comment]
OK,
Let's wait for updating #427 by @Hao-Liu .

File: libvirt/tests/src/virsh_cmd/domain/virsh_domfsfreeze.py [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+
+        if mountpoint:
+            cmd_result = virsh.domfsfreeze(vm_ref, mountpoint=mountpoint,
+                                           options=options,
+                                           unprivileged_user=unprivileged_user,
+                                           uri=uri, debug=True)
+        else:
+            cmd_result = virsh.domfsfreeze(vm_ref, options=options,
+                                           unprivileged_user=unprivileged_user,
+                                           uri=uri, debug=True)
```
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domfsfreeze.py]: The ```if...else``` block is unnecessary.
Just let ```mountpoint = params.get("domfsfreeze_mnt")```.

**Comment by will-Do** [comment]
And, just check the command return value may be not enough for our test, expected more detail checking added.

File: libvirt/tests/src/virsh_cmd/domain/virsh_domfsfreeze.py [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        if not status_error:
+            if cmd_result.exit_status != 0:
+                raise error.TestFail("Fail to do virsh domfsfreeze, error %s" %
+                                     cmd_result.stderr)
+        else:
+            if cmd_result.exit_status == 0:
+                raise error.TestFail("Command 'virsh domfsfreeze' failed ")
+
```
Comment (will-Do) [inline libvirt/tests/src/virsh_cmd/domain/virsh_domfsfreeze.py]: BTW, there's a function ```check_exit_status``` in virttest/utils_test/libvirt.py, you can use it

**Comment by PandaWei** [comment]
@liuzzfnst 
Long time no response.
what's up? do you have time to make some modification as comments?

**Comment by waynesun09** [comment]
The PR have been picked up with new PR:
https://github.com/autotest/tp-libvirt/pull/1202

close this one now, thx

**Pull request closed without merging**